File: samplejava/src/main/java/pyxis/uzuki/live/richutils/samplejava/BaseApplication.java
package pyxis.uzuki.live.richutils.samplejava;

import android.app.Application;

import pyxis.uzuki.live.richutilskt.utils.RichUtils;

/**
 * Created by pyxis on 2017. 7. 13..
 */

public class BaseApplication extends Application {

    @Override
    public void onCreate() {
        super.onCreate();
        RichUtils.initializeFontEngine(this);
    }
}

File: sample/src/main/java/pyxis/uzuki/live/richutilssample/BaseApplication.kt
package pyxis.uzuki.live.richutilssample

import android.app.Application
import pyxis.uzuki.live.richutilskt.utils.initializeFontEngine

/**
 * Created by pyxis on 2017. 7. 13..
 */
class BaseApplication : Application() {

    override fun onCreate() {
        super.onCreate()
        initializeFontEngine()
    }
}